### PR TITLE
[SYTO-1223] Added some tweaks from Kevin's feedback

### PR DESCRIPTION
**Pull request by krlospelaez**
To test:

- Verify tweaks from [Kevin's feedback](https://jira.natgeo.com/browse/SYTO-1223?focusedCommentId=440209&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-440209) are met.

### CODE REVIEW

File: src/styles/theme/captions.html [hunk 1]
```diff
@@ -6,7 +6,7 @@ <h3 class="mt3_heading--h4--2">Photo Captions</h3>
     <div class="mt3_col-lg-8 mt3_col-lg-offset-1">
       <figure>
         <figcaption class="mt3_caption-container--indent">
-          <p class="mt3_caption-body"><span class="mt3_caption-title">Photo Title</span> This is a photo caption. This is a photo caption. This is a photo caption. This is a photo caption. This is a photo caption. This is a photo caption. This is a photo caption. This is a photo caption. This is a photo caption.</p>
+          <p class="mt3_caption-body"><span class="mt3_caption-title">Photo Title</span> This is a Photo Caption. Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes.</p>
```
Comment (hkekeli) [inline src/styles/theme/captions.html]: Could you update the code snippet for this?

Comment (hkekeli) [inline src/styles/theme/captions.html]: Could you update the code snippet for this?

File: src/site/typography/index.html [hunk 2]
```diff
@@ -18,7 +18,7 @@ <h1 class="mt3_heading--h1--2">Typography</h1>
       </code>
     </p>
     <p class="mt3_page-description">
-      to use the fonts in your project, install the main CSS file to your site. Note
+      To use the fonts in your project, install the main CSS file to your site. Note
       that due to license restrictions, fonts will not load outside
       a <strong><code>*.nationalgeographic.com</code></strong> domain. To preview the fonts during local
```
Comment (hkekeli) [inline src/site/typography/index.html]: I'm not seeing the broken characters bug on my local..

Comment (krlospelaez) [inline src/site/typography/index.html]: Me neither, but I updated the specials chars here:

* https://github.com/natgeo/mortar/pull/523/files#diff-34ce191047d4c2175462cefcddcfbd4cR38
* https://github.com/natgeo/mortar/pull/523/files#diff-34ce191047d4c2175462cefcddcfbd4cR51

**Pull request merged**